andreas-antonopoulos/2014-10-08-andreas-antonopolous-canada-senate-bitcoin.es.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Andreas Antonopolous Senado de Canadá Bitcoin (2014-10-08)
+transcript_by: Bryan Bishop 
+translation_by: Blue Moon
+tags: ['security']
```

Comment:
These tags should be translated as well:
s/security/seguridad/

Reply:
The label has been translated.